client/me/site-blocks/main.jsx

```diff
@@ -73,6 +73,9 @@ class SiteBlockList extends Component {
 						{ translate(
 							'Blocked sites will not appear in your Reader and will not be recommended to you.'
 						) }
+						<a href="https://en.support.wordpress.com/reader/#blocking-sites">
+							{ translate( ' Learn more.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( ' Learn more' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add-blocking-support-link)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).